pipeline/scripts/ci/updateConfluence.py

```diff
@@ -0,0 +1,147 @@
+import copy
```

Comment:
It is outside the confluence dir. It will be the middleman connecting the confluence api to groovy. That's why i retained the name as updateConfluence.py. Please let me know if this is not ok.